lib/vmpooler/pool_manager.rb

```diff
@@ -531,11 +531,13 @@ def remove_vmpooler_migration_vm(pool, vm)
       end
     end
 
-    def migrate_vm_and_record_timing(vm_object, vm_name, host, vsphere)
+    def migrate_vm_and_record_timing(vm_object, vm_name, pool, host, source_host_name, dest_host_name, vsphere)
       start = Time.now
       vsphere.migrate_vm_host(vm_object, host)
       finish = '%.2f' % (Time.now - start)
-      $metrics.timing("migrate.#{vm_name}", finish)
+      $metrics.timing("migrate.#{pool}", finish)
```

Comment:
Actually based on looking at how we are getting `vmpooler.destroy` and `vmpooler.clone` metrics elsewhere in this source code it looks like the `$metrics.timing` values do give us `vmpooler.{clone,destroy,migrate}.<pool>.count` already so nevermind.